test/test_ssl.py

```diff
@@ -44,6 +45,15 @@ def test_is_ipaddress_true(self, addr: bytes | str) -> None:
     def test_is_ipaddress_false(self, addr: bytes | str) -> None:
         assert not ssl_.is_ipaddress(addr)
 
+    def test_create_urllib3_context_options(self):
+        context = ssl_.create_urllib3_context()
+        if sys.version_info >= (3, 13):
+            assert context.options & ssl.VERIFY_X509_PARTIAL_CHAIN
+            assert context.options & ssl.VERIFY_X509_STRICT
+        else:
+            assert context.options | ssl.VERIFY_X509_PARTIAL_CHAIN
+            assert context.options | ssl.VERIFY_X509_STRICT
```

Comment:
I think these assert don't check what you indented

```suggestion
            assert not (context.options & ssl.VERIFY_X509_PARTIAL_CHAIN)
            assert not (context.options & ssl.VERIFY_X509_STRICT)
```

Reply:
Indeed, thanks!